addons/addon-base-raas-ui/packages/base-raas-ui/src/models/forms/CreateCidrFormModel.js

```diff
@@ -0,0 +1,51 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+import { createFormSeparatedFormat } from '../../helpers/form';
+
+const createCidrFormModel = ({ existingValues }) => {
+  const fields = ['cidr', 'cidr[].fromPort', 'cidr[].toPort', 'cidr[].protocol', 'cidr[].cidrBlocks'];
+
+  const labels = {
+    'cidr': 'cidr',
+    'cidr[].fromPort': 'From Port',
+    'cidr[].toPort': 'To Port',
+    'cidr[].protocol': 'Protocol',
+    'cidr[].cidrBlocks': 'Allowed CIDR Blocks',
+  };
+
+  const placeholders = {
+    'cidr[].cidrBlocks': 'Enter CIDR ranges here',
+  };
+
+  const extra = {};
+
+  const rules = {
+    'cidr': 'required',
+    'cidr[].fromPort': 'required|integer',
+    'cidr[].toPort': 'required|integer',
+    'cidr[].protocol': 'required',
+    'cidr[].cidrBlocks': 'array',
```

Comment:
Yes, this is the model of the separated properties form that gets created on the UI.

Reply:
Oops sorry, the comment was meant for line 40 for cidrBlocks. I wanted to know why they are not required?

Reply:
Yeah, it should be required as well. I'll test and push that change.

Reply:
If we make it required then users would see an error message when trying to remove all CIDR blocks from an ingress rule

![Screen Shot 2021-01-27 at 5 29 28 PM](https://user-images.githubusercontent.com/20480854/106062749-380afc80-60c5-11eb-8a3e-963d312a6892.png)

Reply:
Interesting! I guess I assumed some cidr range should be allowed otherwise what is the point of having a workspace around. I guess there might be some extreme case when you do this temporarily!

Reply:
Yes, it will always retain the originally configured allowed protocol-port combinations even if there are no CIDR blocks currently in them.

Reply:
Cool thanks for verifying. Resolving this thread.